src/components/SlideButton/SlideButton.scss

```diff
@@ -0,0 +1,74 @@
+@import "../../styles/colors";
+@import "../../styles/typography";
+
+.wrapper {
+  background-color: $white;
+  padding: 16px;
+  width: 100%;
+  box-sizing: border-box;
+}
+.slider-wrapper {
+  position: relative;
+  display: flex;
+  align-items: center;
+  user-select: none;
+  background-color: $blue-500;
+  padding: 4px;
+  height: 48px;
+  border-radius: 28px;
+}
+.slider-track {
+  width: 100%;
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  user-select: none;
+  background-color: $blue-500;
+}
+.overlay {
+  transition: 0.5s ease-out;
+  position: absolute;
+  left: 0px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  user-select: none;
+  background-color: $blue-500;
+}
+.slide-text {
+  user-select: none;
+  width: 100%;
+  text-align: center;
+  font: $laptop-m;
+  color: $white;
+}
+.completed {
+  .slide-text {
+    z-index: 999;
+    transition: 0.5s ease-out;
+  }
+}
+.started {
+  .overlay,
+  .slide-button {
+    transition: none;
+  }
+}
+.slide-button {
+  transition: 0.5s ease-out;
+  position: absolute;
+  left: 0px;
+  width: 46px;
+  height: 46px;
+  border-radius: 23px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #fafafa;
```

Comment:
Please extract this raw hex color value into a well-named local `$` SASS variable at the top of this file.

Reply:
Please make this change ^